### ISSUE TITLE
Active speaker does not work

### ISSUE THREAD

**Issue by mrtordoide** [AgoraIO-Community/Flutter-UIKit#32]
I get this output `Active speaker is disabled` in debug console even when `isActiveSpeakerDisabled` is left to default value. Is there any additional config required?

**Comment by Meherdeep** [comment]
@mrtordoide Please make sure that you're running the latest version of the SDK. Also, note that active speaker functionality is currently available only in the Floating layout. 

If you're still getting an error after these checks, can you please share the logs and flutter doctor.

**Comment by mrtordoide** [comment]
Hi @Meherdeep , yes I have made sure to use latest SDK and layoutType: Layout.floating set. I tried it again with 0.0.3 version of UIKit but it is still showing me this when an active user is identified.

```
I/flutter ( 3974): Local audio state changed state: LocalVideoStreamState.Encoding and error: LocalVideoStreamError.OK
I/flutter ( 3974): Active speaker: 3403
I/flutter ( 3974): Active speaker is disabled
I/flutter ( 3974): Active speaker 3403
```

Below is longer log:

```
W/System.err( 3974): Caused by: android.system.ErrnoException: read failed: EINVAL (Invalid argument)
W/System.err( 3974):    at libcore.io.Linux.readBytes(Native Method)
W/System.err( 3974):    at libcore.io.Linux.read(Linux.java:189)
W/System.err( 3974):    at libcore.io.ForwardingOs.read(ForwardingOs.java:176)
W/System.err( 3974):    at libcore.io.BlockGuardOs.read(BlockGuardOs.java:305)
W/System.err( 3974):    at libcore.io.ForwardingOs.read(ForwardingOs.java:176)
W/System.err( 3974):    at libcore.io.IoBridge.read(IoBridge.java:509)
W/System.err( 3974):    ... 10 more
W/System  ( 3974): A resource failed to call close. 
W/System  ( 3974): A resource failed to call close. 
W/System  ( 3974): A resource failed to call close. 
W/System  ( 3974): A resource failed to call close. 
W/System  ( 3974): A resource failed to call close. 
W/System  ( 3974): A resource failed to call close. 
W/System  ( 3974): A resource failed to call close. 
W/System  ( 3974): A resource failed to call close. 
I/flutter ( 3974): Local audio state changed state: LocalVideoStreamState.Encoding and error: LocalVideoStreamError.OK
I/flutter ( 3974): Active speaker: 3403
I/flutter ( 3974): Active speaker is disabled
I/flutter ( 3974): Active speaker 3403
E/libc    ( 3974): Access denied finding property "net.dns1"
E/libc    ( 3974): Access denied finding property "net.dns2"
E/libc    ( 3974): Access denied finding property "net.dns3"
E/libc    ( 3974): Access denied finding property "net.dns4"
W/System.err( 3974): java.io.IOException: read failed: EINVAL (Invalid argument)
W/System.err( 3974):    at libcore.io.IoBridge.read(IoBridge.java:519)
W/System.err( 3974):    at java.io.FileInputStream.read(FileInputStream.java:320)
W/System.err( 3974):    at sun.nio.cs.StreamDecoder.readBytes(StreamDecoder.java:291)
W/System.err( 3974):    at sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:355)
W/System.err( 3974):    at sun.nio.cs.StreamDecoder.read(StreamDecoder.java:181)
W/System.err( 3974):    at java.io.InputStreamReader.read(InputStreamReader.java:184)
W/System.err( 3974):    at java.io.BufferedReader.fill(BufferedReader.java:172)
W/System.err( 3974):    at java.io.BufferedReader.readLine(BufferedReader.java:335)
W/System.err( 3974):    at java.io.BufferedReader.readLine(BufferedReader.java:400)
W/System.err( 3974):    at io.agora.rtc.gdp.GDPAndroid.readOneLine(GDPAndroid.java:386)
W/System.err( 3974):    at io.agora.rtc.gdp.GDPAndroid.getCpuTemperature(GDPAndroid.java:362)
W/System.err( 3974): Caused by: android.system.ErrnoException: read failed: EINVAL (Invalid argument)
W/System.err( 3974):    at libcore.io.Linux.readBytes(Native Method)
W/System.err( 3974):    at libcore.io.Linux.read(Linux.java:189)
W/System.err( 3974):    at libcore.io.ForwardingOs.read(ForwardingOs.java:176)
W/System.err( 3974):    at libcore.io.BlockGuardOs.read(BlockGuardOs.java:305)
W/System.err( 3974):    at libcore.io.ForwardingOs.read(ForwardingOs.java:176)
W/System.err( 3974):    at libcore.io.IoBridge.read(IoBridge.java:509)
W/System.err( 3974):    ... 10 more
W/System.err( 3974): java.io.IOException: read failed: EINVAL (Invalid argument)
W/System.err( 3974):    at libcore.io.IoBridge.read(IoBridge.java:519)
W/System.err( 3974):    at java.io.FileInputStream.read(FileInputStream.java:320)
W/System.err( 3974):    at sun.nio.cs.StreamDecoder.readBytes(StreamDecoder.java:291)
W/System.err( 3974):    at sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:355)
W/System.err( 3974):    at sun.nio.cs.StreamDecoder.read(StreamDecoder.java:181)
W/System.err( 3974):    at java.io.InputStreamReader.read(InputStreamReader.java:184)
W/System.err( 3974):    at java.io.BufferedReader.fill(BufferedReader.java:172)
W/System.err( 3974):    at java.io.BufferedReader.readLine(BufferedReader.java:335)
W/System.err( 3974):    at java.io.BufferedReader.readLine(BufferedReader.java:400)
W/System.err( 3974):    at io.agora.rtc.gdp.GDPAndroid.readOneLine(GDPAndroid.java:386)
W/System.err( 3974):    at io.agora.rtc.gdp.GDPAndroid.getCpuTemperature(GDPAndroid.java:362)
W/System.err( 3974): Caused by: android.system.ErrnoException: read failed: EINVAL (Invalid argument)
W/System.err( 3974):    at libcore.io.Linux.readBytes(Native Method)
W/System.err( 3974):    at libcore.io.Linux.read(Linux.java:189)
W/System.err( 3974):    at libcore.io.ForwardingOs.read(ForwardingOs.java:176)
W/System.err( 3974):    at libcore.io.BlockGuardOs.read(BlockGuardOs.java:305)
W/System.err( 3974):    at libcore.io.ForwardingOs.read(ForwardingOs.java:176)
W/System.err( 3974):    at libcore.io.IoBridge.read(IoBridge.java:509)
W/System.err( 3974):    ... 10 more
W/System.err( 3974): java.io.IOException: read failed: EINVAL (Invalid argument)
W/System.err( 3974):    at libcore.io.IoBridge.read(IoBridge.java:519)
W/System.err( 3974):    at java.io.FileInputStream.read(FileInputStream.java:320)
W/System.err( 3974):    at sun.nio.cs.StreamDecoder.readBytes(StreamDecoder.java:291)
W/System.err( 3974):    at sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:355)
W/System.err( 3974):    at sun.nio.cs.StreamDecoder.read(StreamDecoder.java:181)
W/System.err( 3974):    at java.io.InputStreamReader.read(InputStreamReader.java:184)
W/System.err( 3974):    at java.io.BufferedReader.fill(BufferedReader.java:172)
W/System.err( 3974):    at java.io.BufferedReader.readLine(BufferedReader.java:335)
W/System.err( 3974):    at java.io.BufferedReader.readLine(BufferedReader.java:400)
W/System.err( 3974):    at io.agora.rtc.gdp.GDPAndroid.readOneLine(GDPAndroid.java:386)
W/System.err( 3974):    at io.agora.rtc.gdp.GDPAndroid.getCpuTemperature(GDPAndroid.java:362)
W/System.err( 3974): Caused by: android.system.ErrnoException: read failed: EINVAL (Invalid argument)
W/System.err( 3974):    at libcore.io.Linux.readBytes(Native Method)
W/System.err( 3974):    at libcore.io.Linux.read(Linux.java:189)
W/System.err( 3974):    at libcore.io.ForwardingOs.read(ForwardingOs.java:176)
W/System.err( 3974):    at libcore.io.BlockGuardOs.read(BlockGuardOs.java:305)
W/System.err( 3974):    at libcore.io.ForwardingOs.read(ForwardingOs.java:176)
W/System.err( 3974):    at libcore.io.IoBridge.read(IoBridge.java:509)
W/System.err( 3974):    ... 10 more
W/System.err( 3974): java.io.IOException: read failed: EINVAL (Invalid argument)
W/System.err( 3974):    at libcore.io.IoBridge.read(IoBridge.java:519)
W/System.err( 3974):    at java.io.FileInputStream.read(FileInputStream.java:320)
W/System.err( 3974):    at sun.nio.cs.StreamDecoder.readBytes(StreamDecoder.java:291)
W/System.err( 3974):    at sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:355)
W/System.err( 3974):    at sun.nio.cs.StreamDecoder.read(StreamDecoder.java:181)
W/System.err( 3974):    at java.io.InputStreamReader.read(InputStreamReader.java:184)
W/System.err( 3974):    at java.io.BufferedReader.fill(BufferedReader.java:172)
W/System.err( 3974):    at java.io.BufferedReader.readLine(BufferedReader.java:335)
W/System.err( 3974):    at java.io.BufferedReader.readLine(BufferedReader.java:400)
W/System.err( 3974):    at io.agora.rtc.gdp.GDPAndroid.readOneLine(GDPAndroid.java:386)
W/System.err( 3974):    at io.agora.rtc.gdp.GDPAndroid.getCpuTemperature(GDPAndroid.java:362)
W/System.err( 3974): Caused by: android.system.ErrnoException: read failed: EINVAL (Invalid argument)
W/System.err( 3974):    at libcore.io.Linux.readBytes(Native Method)
W/System.err( 3974):    at libcore.io.Linux.read(Linux.java:189)
W/System.err( 3974):    at libcore.io.ForwardingOs.read(ForwardingOs.java:176)
W/System.err( 3974):    at libcore.io.BlockGuardOs.read(BlockGuardOs.java:305)
W/System.err( 3974):    at libcore.io.ForwardingOs.read(ForwardingOs.java:176)
W/System.err( 3974):    at libcore.io.IoBridge.read(IoBridge.java:509)
W/System.err( 3974):    ... 10 more
I/flutter ( 3974): Remote audio state changed for 3403, state: AudioRemoteState.Stopped and reason: AudioRemoteStateReason.RemoteMuted
I/flutter ( 3974): Remote audio state changed for 3403, state: AudioRemoteState.Decoding and reason: AudioRemoteStateReason.RemoteUnmuted
I/flutter ( 3974): Remote audio state changed for 3403, state: AudioRemoteState.Starting and reason: AudioRemoteStateReason.Internal
```

**Comment by tadaspetra** [comment]
Active Speaker was fixed in version 0.0.3. If you are still experiencing the issue feel free to reopen
